image/cli/mascli/templates/gitops/appset-configs/cluster/instance/ibm-spss.yaml.j2

```diff
@@ -7,4 +7,6 @@ ibm_spss:
   cpd_service_storage_class: "{{ CPD_SERVICE_STORAGE_CLASS }}"
   spss_channel: "<path:{{ SECRETS_PATH }}:{{ SECRET_KEY_SPSS_CHANNEL }}>"
   spss_version: "<path:{{ SECRETS_PATH }}:{{ SECRET_KEY_SPSS_VERSION }}>"
+  spss_install_plan: "{{ SPSS_INSTALL_PLAN }}"
   canvasbase_channel: "<path:{{ SECRETS_PATH }}:{{ SECRET_KEY_CANVASBASE_CHANNEL }}>"
+  canvasbase_install_plan: "{{ CANVASBASE_INSTALL_PLAN }}"
```

Comment:
canvasbase_install_plan is not set by anything.

Reply:
Done